fairscale/nn/data_parallel/sharded_ddp.py

```diff
@@ -39,7 +39,7 @@ class ShardedDataParallel(nn.Module):
             distributed gradient reduction. If None, the default WORLD process group
             will be used.
         buffer_size (int, optional): number of elements to buffer before
-            performing reduce (default: 256M). Used to reduce multiple small
+            performing reduce (default: 1M). Used to reduce multiple small
```

Comment:
Curious, how is 1M default determined?

Reply:
I think that it's something we should tweak probably, I just changed it after a discussion with @msbaines who was mentioning batching kb params, and because on 4 gpus and training a RN101 the zero savings are around 400MB, so buffering 256MB is probably not worth it.